### PR TITLE
Add limit switch to intake

### LINKED ISSUE #104: Add limit switch to intake
Should stop motors when triggered

### PR DESCRIPTION
**Pull request by KaneBarnes**
Added limit switch for intake motors and changed charger tread motor naming to be more specific
Closes #104

### CODE REVIEW

File: src/main/java/frc/robot/RobotMap.java [hunk 1]
```diff
@@ -27,8 +27,8 @@ public static final class mapDrivetrain {
 
   public static final class mapChargerTreads {
 
-    public static final int LEFT_MOTOR_CAN = 8;
-    public static final int RIGHT_MOTOR_CAN = 9;
+    public static final int TREADS_LEFT_MOTOR_CAN = 8;
+    public static final int TREADS_RIGHT_MOTOR_CAN = 9;
```
Comment (IanSzalai) [inline src/main/java/frc/robot/RobotMap.java:31]: these are not related to this pr

**Review by ACat701 — APPROVED** [review]
👍
